sdc/tests/test_series.py

```diff
@@ -6007,39 +6007,39 @@ def test_series_islower_str(self):
     @skip_sdc_jit("Series.str.isalnum is not supported yet")
     def test_series_isalnum_str(self):
         cfunc = self.jit(isalnum_usecase)
-        test_data = test_global_input_data_unicode_kind4
+        test_data = test_global_input_data_unicode_kind4 + test_global_input_data_unicode_kind1
```

Comment:
I think the following way is more correct:
`test_data = [test_global_input_data_unicode_kind4, test_global_input_data_unicode_kind1]`

Reply:
Could you please use `test_global_input_data_unicode_kind1` and `test_global_input_data_unicode_kind4` from `common_functions.py`?